packages/slice-machine/src/features/onboarding/SharedOnboardingGuide.tsx

```diff
@@ -0,0 +1,26 @@
+import { OnboardingGuide } from "@prismicio/editor-fields";
+import { useMediaQuery } from "@prismicio/editor-ui";
+
+import { telemetry } from "@/apiClient";
+
+import { useOnboarding } from "./useOnboarding";
+
+export function SharedOnboardingGuide() {
+  const isOnboardingVisible = useMediaQuery({ min: "medium" });
```

Comment:
💡 Why not having all conditions of the onboarding card display in the parent `packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx`? So it's easier to know what conditions handle the display?

Here it's duplicated in both onboarding guide card, shared or not.